tests/application/static_site/test_views.py

```diff
@@ -101,6 +105,107 @@ def test_departmental_user_cannot_see_page_unless_shared(test_app_client, logged
     assert page.h1.text.strip() == "Test Measure Page"
 
 
+def test_departmental_user_cannot_see_draft_measure(test_app_client, logged_in_dept_user):
+    measure_version = MeasureVersionFactory(title="Test Measure Page", measure__shared_with=[], status="DRAFT")
+
+    resp = test_app_client.get(
+        url_for(
+            "static_site.measure_version",
+            topic_slug=measure_version.measure.subtopic.topic.slug,
+            subtopic_slug=measure_version.measure.subtopic.slug,
+            measure_slug=measure_version.measure.slug,
+            version=measure_version.version,
+        )
+    )
+
+    assert resp.status_code == 403
+
+
+def test_departmental_user_can_see_published_measure(test_app_client, logged_in_dept_user):
+    measure_version = MeasureVersionFactory(title="Test Measure Page", measure__shared_with=[], status="APPROVED")
+
+    resp = test_app_client.get(
+        url_for(
+            "static_site.measure_version",
+            topic_slug=measure_version.measure.subtopic.topic.slug,
+            subtopic_slug=measure_version.measure.subtopic.slug,
+            measure_slug=measure_version.measure.slug,
+            version=measure_version.version,
+        )
+    )
+
+    assert resp.status_code == 200
+    page = BeautifulSoup(resp.data.decode("utf-8"), "html.parser")
+    assert page.h1.text.strip() == "Test Measure Page"
+
+
+def test_departmental_user_can_see_latest_published_version(test_app_client, logged_in_dept_user):
+
+    measure = MeasureFactory(shared_with=[])
+
+    MeasureVersionFactory(title="Old Test Measure Page", status="APPROVED", measure=measure, version="1.0")
+    MeasureVersionFactory(title="Updated Test Measure Page", status="DRAFT", measure=measure, version="2.0")
+
+    resp = test_app_client.get(
+        url_for(
+            "static_site.measure_version",
+            topic_slug=measure.subtopic.topic.slug,
+            subtopic_slug=measure.subtopic.slug,
+            measure_slug=measure.slug,
+            version="latest",
+        )
+    )
+
+    assert resp.status_code == 200
+    page = BeautifulSoup(resp.data.decode("utf-8"), "html.parser")
+    assert page.h1.text.strip() == "Old Test Measure Page"
+
+
+def test_departmental_user_can_see_latest_draft_version_if_they_have_access(test_app_client, logged_in_dept_user):
+
+    measure = MeasureFactory(shared_with=[])
```

Comment:
Same here, you should be able to do `MeasureVersionFactory(measure__shared_with=[logged_in_dept_user])`